pkg/costmodel/cluster.go

```diff
@@ -216,10 +216,20 @@ func ComputeClusterCosts(client prometheus.Client, provider cloud.Provider, wind
 		resChs = append(resChs, bdResChs...)
 	}
 
+	resDataCount, _ := resChs[0].Await()
+	resTotalGPU, _ := resChs[1].Await()
+	resTotalCPU, _ := resChs[2].Await()
+	resTotalRAM, _ := resChs[3].Await()
+	resTotalStorage, _ := resChs[4].Await()
+	resTotalLocalStorage, _ := resChs[5].Await()
+	if ctx.HasErrors() {
```

Comment:
Note to self: after [PR 502](https://github.com/kubecost/cost-model/pull/502/files) goes in, update to match